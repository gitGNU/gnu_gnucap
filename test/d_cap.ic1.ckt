' cap with initial condition, ignored because I didn't say "uic"
c1 1 0 1 ic=3
r1 1 0 1
.list
.print tran v(1)
.tran 1 10 0
.end
