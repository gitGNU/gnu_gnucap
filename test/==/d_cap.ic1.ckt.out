' cap with initial condition, ignored because I didn't say "uic" 
C1  1  0   1.  ic= 3.
R1  1  0  1.
#Time       v(1)      
 0.         0.        
 1.         0.        
 2.         0.        
 3.         0.        
 4.         0.        
 5.         0.        
 6.         0.        
 7.         0.        
 8.         0.        
 9.         0.        
 10.        0.        
